src/nullablevector.jl

```diff
@@ -311,3 +313,27 @@ function Base.empty!(X::NullableVector)
     empty!(X.isnull)
     return X
 end
+
+function vcat(A::AbstractVector, B::AbstractVector...)
```

Comment:
I really don't know. Clearly we can't reach a satisfying state without improvements in Julia. But which of the partial solutions is better, I don't know...